Flapjack/CoreData/Extensions/Dictionary+Extensions.swift

```diff
@@ -8,13 +8,18 @@
 
 import Foundation
 
-internal extension Dictionary where Key == String, Value == Any {
+internal extension Dictionary where Key == String, Value == Optional<Any> {
```

Comment:
```suggestion
internal extension Dictionary where Key == String, Value == Any? {
```
Would you be open to changing this so the linter is happier?

Reply:
Oh for sure! Lemme take a gander